packages/webapp/src/locales/i18n.js

```diff
@@ -25,6 +25,7 @@ i18n
     backend: {
       queryStringParams: { v: APP_VERSION },
     },
+    ns: ['crop', 'expense', 'task'],
```

Comment:
Some i18n.t() in util files stopped working for some reason, and I had to add namespaces.
https://www.i18next.com/how-to/add-or-load-translations#combined-with-a-backend-plugin